templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -8,7 +8,7 @@ contents: |
   Wants=NetworkManager-wait-online.service
   After=firstboot-osupdate.target
   After=NetworkManager-wait-online.service openvswitch.service network.service nodeip-configuration.service
-  Before=kubelet-dependencies.target node-valid-hostname.service
+  Before=kubelet-dependencies.target node-valid-hostname.service dnsmasq.service
```

Comment:
As long as this does not break any of the e2e tests (and maybe we should run some payloads?) this makes sense.

Reply:
Sure, I'll run the same payload jobs that ran on the original change as soon as the standard pre-submits complete.

Reply:
to reduce potential impact, would you consider it acceptable if the Before dnsmasq.service only applies to Azure? i.e. add this to Azure templates instead?

Reply:
Ok, I was originally concerned that it would introduce risk of divergent changes, but looks like this file is pretty stable as it hasn't changed in three years. Updated the PR.